cmd/utils/flags.go

```diff
@@ -908,6 +908,18 @@ var (
 		Category: flags.RollupCategory,
 		Value:    true,
 	}
+	RollupSequencerEnableTxConditionalFlag = &cli.BoolFlag{
+		Name:     "rollup.sequencerenabletxconditional",
+		Usage:    "Serve the eth_sendRawTransactionConditional endpoint and apply the conditional constraints on mempool inclusion & block building",
+		Category: flags.RollupCategory,
+		Value:    false,
+	}
+	RollupSequencerTxConditionalRateLimitFlag = &cli.Uint64Flag{
```

Comment:
Is this flag used? see https://github.com/ethereum-optimism/op-geth/pull/330/files#r1690167464

Reply:
yea this flag is consumed in the miner config. It sets `RollupTransactionConditionalBurstRate` in the miner config which is forwarded to the miner rater limiter.

By having a rate limiter in the miner, it also allows a chain operator to safely expose this endpoint publicly without the proxy if they choose to

Reply:
I don't buy that this rate limiter would actually make this safe to expose to the public internet, if a real ddos attack makes it to userland then you are ngmi

Reply:
yea I agree it's not sufficient but provides some level of protection to bursts of conditional txs